src/Microsoft.CSharp/src/Microsoft/CSharp/RuntimeBinder/Semantics/Symbols/PropertySymbol.cs

```diff
@@ -31,5 +31,7 @@ public IndexerSymbol AsIndexerSymbol()
             Debug.Assert(isIndexer());
             return (IndexerSymbol)this;
         }
+
+        public bool Bogus { get; set; }
```

Comment:
I suggest keeping with other fields, or converting all fields to properties.
I know there is a bigger effort of refactoring these things, but we shouldn't special case this here now.